module/canvas/tree/talent-tree-talent.mjs

```diff
@@ -85,6 +85,7 @@ export default class CrucibleTalentTreeTalent extends CrucibleTalentIcon {
   /* -------------------------------------------- */
 
   #onPointerOver(event) {
+    if ( document.elementFromPoint(event.globalX, event.globalY)?.id !== "crucible-talent-tree" ) return;
```

Comment:
Fine merging this for now, but I wonder if it's possible to retrieve this directly from `event.data.originalEvent` or something.